monorepo-builder.php

```diff
@@ -11,6 +11,7 @@
         __DIR__ . '/src/core',
         __DIR__ . '/src/adapter',
         __DIR__ . '/src/lib',
+        __DIR__ . '/src/web',
```

Comment:
hmm the way I was thinking about it was that we are going to build static content and only that static content would be published to standalone repo that would be gh static page. 

I'm also not sure if we even want to keep the same composer.json for landing page, it's not that this landing page should use the same dependencies and the code, no?